cegs_portal/search/templates/search/v1/partials/_features.html

```diff
@@ -1,5 +1,93 @@
+{% load humanize %}
+<style>
```

Comment:
Please don't put styling information in partials. It can cause funny issues with its included in page because the styles aren't "scoped" to the partial -- they can make global changes to other parts of the page it's included in.

It also looks like a lot of this is duplicated in the style tag of reg_effect.html

Reply:
Was that done today? We were going to clean up CSS from partials etc in the near future. Did you do that already?